pkgs/build-support/build-fhsenv-bubblewrap/default.nix

```diff
@@ -233,7 +232,7 @@ in runCommandLocal name {
     '';
     inherit args fhsenv;
   };
-} ''
+}) ''
   mkdir -p $out/bin
   ln -s ${bin} $out/bin/${pname}
```

Comment:
Ultimately, this line is why we cannot get rid of `pname = args.pname or (lib.getName args.name)`. The fix is to decouple the binary name from the package name but that is way out of scope for this PR

Reply:
No. We can get rid of it right now. If you're explicitly passing a `name` with a version in it, you will get an executable with exactly that name; that's how it should work.

If you've got pname and version, you should simply pass those and you will get "correct" behaviour.

We should not add code for potential wrong usages that violates the principle of least surprise. It's conceivable for example that an intended package name may look like a version or that a user might want the executable to be versioned. Those use-cases should not be forbidden because someone might perhaps use it wrong.

Reply:
Great. With https://github.com/NixOS/nixpkgs/pull/271071#discussion_r1412737161 answered the fix becomes quite simple.